x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -162,10 +162,28 @@ describe('Resolver, when analyzing a tree that has two related events for the or
           button.simulate('click');
         }
       });
-      it('should open the submenu', async () => {
+      it('should open the submenu and display exactly one option with the correct count', async () => {
         await expect(
           simulator.map(() => simulator.processNodeSubmenuItems().map((node) => node.text()))
         ).toYieldEqualTo(['2 registry']);
+        await expect(
```

Comment:
The above `expect` already asserts that there is 1 item. is this extra expectation needed?

Reply:
That being said, i think the change to the test name is a good one.

Reply:
it's a predicate to the one below (that expects 0). Just an extra check and maybe it helps a reader see more quickly (click once -> expect 1 -> click again -> expect 0)